### PR TITLE
chore(PPDSC-2848): remove ecr and python in circleci config

### PR DESCRIPTION
**Pull request by Xin00163**
PPDSC-2848

-Keep /terraform/ecr (for now to aid destruction of resources)
- Remove ECR references in circle yaml (build_push_docker_ecr, deploy_terraform_ecr_dev, build_push_docker_ecr_release)
- Rename aws_init to init and keep bare-minimum. Remove the current two steps in it and any references to Python in it, as no longer need Python to install was-cli in a node container.

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2848-remove-ecr-and-python
  Storybook: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2848-remove-ecr-and-python/storybook

File: .circleci/config.yml [hunk 1]
```diff
@@ -91,11 +91,6 @@ executors:
     docker:
       - image: hashicorp/terraform:0.12.28
```
Comment (JohnTParsons) [inline .circleci/config.yml:92]: Is it now safe to remove this Terraform-only docker image or is there something still using it?

Comment (Xin00163) [reply .circleci/config.yml:92]: It's still used in `deploy_terraform` and that is used in deploying `package: dns`. Not sure if `dns` is still in use?

Comment (JohnTParsons) [reply .circleci/config.yml:92]: Please can you remove any steps to DNS as that was only needed to support EKS and then remove deploy_terraform and that Docker image

**Review by JohnTParsons — APPROVED** [review]
Nice work, Xin

**Review by Vanals — APPROVED** [review]

**Pull request merged**